app/views/tmdb/actor_search.html.erb

```diff
@@ -3,8 +3,17 @@
 <h1>Search by Actor</h1>
 <% if @results&.actor.blank? %> <!-- Display the search form -->
   <%= form_tag '/tmdb/actor_search', { class: "form-class", id: "actor-search", role: "form", method: :get } do %>
-    <%= text_field_tag :actor, nil, class: "form-control search-form autocomplete-search-field", id: "actor_name_actor_search", placeholder: "Enter Actor Name", data: { autocomplete_source: person_autocomplete_path } %>
-    <%= select_tag :sort_by, options_for_select(MovieDataService::SORT_BY), {:prompt => "Sort by...", class: "form-control search-form", :include_blank => false } %>
+    <%= text_field_tag :actor, nil,
+      class: "form-control search-form autocomplete-search-field",
+      id: "actor_name_actor_search",
+      required: true,
```

Comment:
Here I've added `required: true` and broken the `text_field_tag` onto multiple lines. This is the same pattern I've done in all 5 search views.

Reply:
nice! 🚀